app/lib/index.js

```diff
@@ -99,6 +100,10 @@ rpc.on('update available', ({ releaseName, releaseNotes }) => {
   store_.dispatch(updaterActions.updateAvailable(releaseName, releaseNotes));
 });
 
+rpc.on('move', () => {
+  store_.dispatch(uiActions.setFontSmoothing());
```

Comment:
Thanks, this has been a really well-done project, and is fun to work on! I added a `UI_WINDOW_MOVE` action in a way that I think does what you were asking for. If not, let me know 😄